### PR TITLE
Updated vocabularies and content types for using urgency field.

### PR DESCRIPTION
**Pull request by eozan**
SDPA-76
SDPA-77

### CODE REVIEW

**Comment by vied12** [comment]
:+1:

File: server/settings.py [hunk 1]
```diff
@@ -97,3 +97,5 @@
     'pa.topics',
     'pa.pa_img',
 ]
+
+DEFAULT_URGENCY_VALUE_FOR_MANUAL_ARTICLES = None
```
Comment (actionless) [inline server/settings.py]: should be wraped inside `env` function, see above in this module

Comment (petrjasek) [inline server/settings.py]: it shouldn't, it's client specific config

**Comment by actionless** [comment]
:+1: then

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**